pkg/descheduler/node/node.go

```diff
@@ -121,3 +123,23 @@ func IsNodeUschedulable(node *v1.Node) bool {
 	}
 	return false
 }
+
+func PodFitAnyNode(pod *v1.Pod, nodes []*v1.Node) bool {
+	for _, node := range nodes {
+		nodeInfo := schedulercache.NewNodeInfo()
+		nodeInfo.SetNode(node)
+
+		ok, _, err := predicates.PodMatchNodeSelector(pod, nil, nodeInfo)
```

Comment:
@containscafeine - I believe you want to reuse predicate functions from scheduler code, I think its better not to use them at this point as they are not ready to be used for this purpose. The easiest way is to replicate the podMatchNodeSelector function in here instead of importing it predicates file and we can remove vendor dependencies.

Reply:
ACK, I've copied the relevant code to descheduler itself.